fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -2699,27 +2714,6 @@ ACTOR void setupAndRun(std::string dataFolder,
 	state Optional<TenantName> defaultTenant;
 	state Standalone<VectorRef<TenantNameRef>> tenantsToCreate;
 	state TenantMode tenantMode = TenantMode::DISABLED;
```

Comment:
can we set this to an INVALID value to start with? The code behavior is updated by this PR, earlier the mode would always be valid in `setupSimulatedSystem`, however, now the method is responsible to setup the right tenantMode. 
Maybe am paranoid, but how about:
1. Set value as `invalid` to start with
2. Assert valid value after `setupSimulatedSystem`

Reply:
reasonable. will update.